firmware/integration/rusefi_config.txt

```diff
@@ -981,6 +981,7 @@ bit skippedWheelOnCam,"On camshaft","On crankshaft";Where is your primary skippe
 	bit complexWallModel,"Advanced (tables)","Basic (constants)";Should we use tables to vary tau/beta based on CLT/MAP, or just with fixed values?
 	bit alwaysInstantRpm
 	bit isMapAveragingEnabled
+	bit overrideCrankingTaperDurationSetting; If enabled, use separate temperature multiplier table for cranking taper duration.
```

Comment:
I'm not sure `override` is the right language here - maybe `useCrankingIdleTaperTable` or something would be clearer?

Reply:
Will change, just looked add `overrideCrankingIacSetting`